src/mscorlib/shared/System/Runtime/Versioning/TargetFrameworkAttribute.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-//
```

Comment:
I have been reconciling diffs between CoreCLR and CoreRT, not much beyond that. The purpose comment was on both sides so I have kept it.

We have the purpose comment in number of files in the shared partition. I have deleted it only if it was deleted in CoreRT and it had zero information value.